server/api/expense-pool/expense-pool.controller.js

```diff
@@ -21,65 +21,30 @@ controller.addPool = (req, res) => {
   .catch(error => handleError(res, error));
 };
 
-
 controller.addUserToPool = (req, res) => {
```

Comment:
Hmm I'm not sure which commented out ExpensePool you are referring to. 

If you mean why the look up for expense pool is removed. All of these routes dealing with a specific expense pool are behind the `isPoolMember` authorization middleware the `req` will always have a `expensePool` property with the expense pool specified by the `req.params.id`. That means we don't need to look up the expense pool in these controller functions.